### PR TITLE
EES-1407 UI tests

### PR DESCRIPTION
**Pull request by luke-h1**
* This PR refactors `publish_release_and_amend_2.robot` to use more filters as stated in `EES-1407`. Instead of using Dates.csv as the initial file it now uses `seven_filters.csv` which allows the test to utilize a lot more filters than dates.csv. Two new files have been added to the files/ directory (`seven_filters.csv & seven_filters.meta.csv` ).

*  Initially this test seemed to have lots of issues with elements being rendered (not sure if this was due to react lazy loading components or just a quirk of selenium) which have now been removed to speed up to the time it takes to run on the UI test pipeline. 

* In addition this test case does the following: 
   * creating a permalink & checking for error alert (`create_amendment_check_permalink.robot `has been removed)
   * creating and editing a methodology (see https://github.com/dfe-analytical-services/explore-education-statistics/pull/2149)
   * validating table cells / columns before and after an amendment
   * checking deleted subject files do not appear after an amendment

### CODE REVIEW

File: tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot [hunk 1]
```diff
@@ -8,25 +8,83 @@ Suite Setup       user signs in as bau1
...
 
 *** Variables ***
-${DETAILS_HEADING}  Academic Year 2020/21
-${PUBLICATION_NAME}  Publish Release and Amend 2 Publication %{RUN_IDENTIFIER}
-${SUBJECT_NAME}  Dates test subject
-${SECOND_SUBJECT}  Upload file test subject
-
+${RELEASE_NAME}  Academic Year Q1 2020/21
+${PUBLICATION_NAME}  UI tests - publish release and amend 2 %{RUN_IDENTIFIER}
+${SUBJECT_NAME}  seven filters
```
Comment (ntsim) [inline tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot:13]: Would upper case

File: tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot [hunk 2]
```diff
@@ -8,25 +8,83 @@ Suite Setup       user signs in as bau1
...
+${SECOND_SUBJECT}  dates subject
```
Comment (ntsim) [inline tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot:14]: Same as above, would uppercase this

File: tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot [hunk 3]
```diff
@@ -161,20 +312,37 @@ Go to permalink
...
 
+Edit methodology
+    [Tags]  HappyPath
+    user clicks link  manage methodologies
+    user clicks element  //*[@id="approved-methodologies-tab"]
+    user clicks link  ${METHODOLOGY_NAME}
+    user clicks link  Manage content
+    user clicks button  ${METHODOLOGY_NAME}
+    user clicks button  Add text block
+    user waits until page contains button  Edit block
```
Comment (ntsim) [inline tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot:329]: Do we need this wait?

File: tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot [hunk 4]
```diff
@@ -102,50 +161,142 @@ Select "Pupil Exclusions" publication
...
+    # 5 YEAR AFTER STUDY
+    user checks results table row heading contains  29  2  Asian/Asian British
+    user checks results table row heading contains  30  1  Black/African/Caribbean/Black British
+    user checks results table row heading contains  31  1  Mixed/Multiple ethnic group
+    user checks results table row heading contains  32  1  Not Known/Not Provided
+    user checks results table row heading contains  33  1  Other Ethnic Group
+    user checks results table row heading contains  34  1  Total
+    user checks results table row heading contains  35  1  White
+
+Validate table cells
```
Comment (ntsim) [inline tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot:277]: How big is this table? I'm a little concerned that it's too big to reasonably write all the assertions for and we're leaving ourselves open for potential bugs due to not covering the entire table.

I would consider cutting back the selections to something where we can test the entire table properly

**Comment by luke-h1** [comment]
Created Jira ticket to keep track of the issue whereby the table would render differently on each test run. In addition to this I've refactored the test to use a more reliable table that doesn't change on each test run and added some checks in to make sure that the Set Page View div doesn't interrupt selenium editing a methodology

**Review by ntsim — APPROVED** [review]

**Pull request merged**